packages/nectar/src/sagas/errors/handleErrors/handleErrors.saga.ts

```diff
@@ -23,14 +23,27 @@ function* retryRegistration(communityId: string) {
   log(`registering certificate for community ${communityId} failed, trying again`)
 }
 
+let registrationAttempts: number = 0
+
 export function* handleErrorsSaga(
   action: PayloadAction<ReturnType<typeof errorsActions.addError>['payload']>
 ): Generator {
   const error: ErrorPayload = action.payload
   if (error.type === SocketActionTypes.REGISTRAR) {
-    if (error.code === ErrorCodes.NOT_FOUND) {
-      yield* call(delay, 15000)
-      yield* call(retryRegistration, error.community)
+    if (error.code === ErrorCodes.FORBIDDEN) {
+      yield* put(errorsActions.addError(error))
+    }
+    if (error.code === ErrorCodes.NOT_FOUND || error.code === ErrorCodes.SERVER_ERROR || error.code === ErrorCodes.SERVICE_UNAVAILABLE) {
+      if (registrationAttempts < 7) {
```

Comment:
Why 7?

Reply:
There is near 99.9% chance that if registrar is online it will succeed in 7 attempts. I think 5 may be risky and 10 may be too long? That's what we decided on call yesterday, to make it 99.9% that it will succeed, and also 7 is a great number.

Reply:
I would add a comment why. I don't like mysterious numbers in code 😅